app/src/main/java/mozilla/lockbox/store/AutoLockStore.kt

```diff
@@ -65,10 +68,11 @@ class AutoLockStore(
 
     override fun injectContext(context: Context) {
         preferences = PreferenceManager.getDefaultSharedPreferences(context)
+        (lockRequired as Subject).onNext(lockCurrentlyRequired())
```

Comment:
`lockRequired` should perhaps be a `PublishSubject` called `autolockTriggered`. We discussed this in person, but I don't know who should do the work — here or another PR. :)